3rdparty/3rdparty.pri

```diff
@@ -30,8 +30,10 @@ if(win32*):exists( $$PWD/qredisclient/qredisclient.lib ) {
 include($$PWD/pyotherside.pri)
 
 #LZ4
-LZ4DIR = $$PWD/lz4/
-INCLUDEPATH += $$LZ4DIR/lib
+!defined(SYSTEM_LZ4, var) {

```

Comment:
No need to modify these lines. Just leave it as is because its common code for all platforms.